alphadia/transferlearning/train.py

```diff
@@ -5,6 +5,7 @@
 import torch
```

Comment:
should we also take action in peptdeep? e.g. log a warning if the precursors are not sorted?